docs/source/reference/config.rst

```diff
@@ -224,7 +224,7 @@ Available fields and semantics:
     # LOCAL_CREDENTIALS: The user's local credential files will be uploaded to
     # GCP instances created by SkyPilot. They are used for accessing cloud
     # resources (e.g., private buckets) or launching new instances (e.g., for
-    # spot/serve controllers).
+    # job/serve controllers).
```

Comment:
I was still calling it `job controller` because grammatically we normally do not use plural for an attributive noun, but after looking around, it seems fine to call it `jobs controller` as well.